supporting_content/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,55 @@
+# Column: CommitmentDiscountType
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column                    |  Example Values  |
+|----------|--------------------------|---------------------------|------------------|
+| AWS | CUR                      | Not available   | N/A |
+| Google Cloud | BigQuery Billing Export | credit.type              | COMMITTED_USAGE_DISCOUNT, COMMITTED_USAGE_DISCOUNT_DOLLAR_BASE |
+| Microsoft | Cost Details             | Not available               | N/A |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | CommitmentDiscountType     | CommitmentDiscountProgram (Name TBD)     |
+|----------|--------------------------|----------------------------|------------------------------------------|
+| AWS | CUR (PurchaseOption)                   | Usage-Based                | Reserved Instances              |
+| AWS | CUR (PurchaseOption)                   | Usage-Based                | Savings Plans                   |
+| Google Cloud | BigQuery Billing Export | Usage-Based        | Resource-based CUD                            |
+| Google Cloud | BigQuery Billing Export | Spend-Based        | Spend-based CUD                               |
+| Microsoft | Cost Details (PricingModel)| Usage-Based                     | Savings Plan                     |
+| Microsoft | Cost Details (PricingModel)| Usage-Based                     | Reservation                      |
```

Comment:
Aren't AWS and Azure Savings plans examples for spend-based commitments?
Aren't AWS and Azure Savings plans examples for spend-based commitments? 
The commitment is to spend hourly a certain amount, Although how the discount is applied and resulting savings depend on what kind of resources were running, the commitment is to spend hourly a certain amount.

Reply:
Thanks - this was an oversight - updated